packages/webapp/cypress/e2e/tasksFlow.spec.js

```diff
@@ -0,0 +1,750 @@
+import { getDateInputFormat } from '../../src/util/moment';
+
+let userEmail;
+let userPassword;
+let Data;
+let fullName;
+
+before(() => {
+  cy.getEmail().then((email) => {
+    userEmail = email;
+    cy.log(userEmail);
+
+    cy.getPassword().then((newpassword) => {
+      userPassword = `${newpassword}+1@`;
+
+      cy.fixture('tasks').then((data) => {
+        Data = data;
+      });
+      cy.visit('/');
+      cy.get('[data-cy=email]', { timeout: 60 * 1000 }).should('exist');
+      cy.get('[data-cy=continue]').should('exist');
+      cy.get('[data-cy=continue]').should('be.disabled');
+      cy.get('[data-cy=continueGoogle]').should('exist');
+
+      //create test data
+      const emailOwner = userEmail;
+      const usrname = emailOwner.indexOf('@');
+      const emailWorker = emailOwner.slice(0, usrname) + '+1' + emailOwner.slice(usrname);
+      const gender = 'Male';
+      fullName = 'Test Farmer';
+      const password = userPassword;
+      const farmName = 'UBC FARM';
+      const location = '49.250833, -123.2410777';
+      const fieldName = 'Test Field';
+      const workerName = 'John Worker';
+      const testCrop = 'New Crop';
+      const role = 'Manager';
+      const lang = 'English';
+      let taskType_id;
+
+      const date = new Date();
+      //Inputs for crop plan
+      const daysGermination = 10;
```

Comment:
A lot of these constants can be and are already imported with cy.fixtures I believe